bin2tif/readme.md

```diff
@@ -66,3 +141,4 @@ qsub /projects/arpae/terraref/shared/extractors/extractors-stereo-rgb/bin2tif/ba
 * All the Python scripts syntactically support Python 2.7 and above. Please make sure that the Python in the running environment is in appropriate version.
```

Comment:
@ZongyangLi does this refer to all of the code in here? Does it imply it works with Python 3.x?

Reply:
It should be work for Python 3.x, but Python 2.7 is recommended.